lib/tasks/fixes.rake

```diff
@@ -160,6 +160,35 @@ namespace :fixes do
     update_event_submission_version(submission_3_id, "e906d7be-0091-44dc-9d88-b4692b5e89e4", 5)
   end
 
+  desc "Append high_value flag to submissions"
+  task adds_high_value: :environment do
+    # Query to select all submission versions that doesn't have a high_value flag
+    # and go through each submission version to add the high_value flag
+    # and show logs for total affected submission versions
+    # and show logs for successful submission versions
+    # and show logs for failed submission versions
+    submission_versions = SubmissionVersion.where("application -> 'cost_summary' ->> 'high_value' IS NULL")
+    total_affected_submission_versions = submission_versions.count
+    successful_submission_versions = 0
+    failed_submission_versions = 0
+    submission_versions.each do |submission_version|
```

Comment:
It's more performant to do `find_each` than `each`, because the former batch loads them from the DB rather than pulling them all into memory from a single query